src/components/mgt-people-picker/mgt-people-picker.ts

```diff
@@ -125,12 +125,16 @@ export class MgtPicker extends MgtTemplatedComponent {
     if (event.keyCode == 40 || event.keyCode == 38) {
       //keyCodes capture: down arrow (40) and up arrow (38)
       this.handleArrowSelection(event);
-      event.preventDefault();
+      if (this._userInput.length > 0) {
```

Comment:
This seems like it's fixing a different bug - if so, please add it to the description of the PR for tracking

Reply:
This is weird, but its actually for accessibility. This is because the input of the people picker was using the prevent default so that you could "tab" and select a person, and then it will focus on the input immediately after, rather than somewhere else on the page.

This is an issue for any "tab" navigation, so when there is no input, you can actually navigate.